daphne/cli.py

```diff
@@ -208,15 +213,16 @@ def run(self, args):
         # Decode args
         args = self.parser.parse_args(args)
         # Set up logging
-        logging.basicConfig(
-            level={
-                0: logging.WARN,
```

Comment:
This is a change in behaviour... `verbosity=0` leads to `logging.WARN` currently.